src/util/Constants.ts

```diff
@@ -317,6 +317,9 @@ export const CommandResponses = {
 					`> Total Boosts: ${guild.premiumSubscriptionCount ? `${guild.premiumSubscriptionCount} ${plural('boost', guild.premiumSubscriptionCount > 1)}` : 'None'}`
 				],
 				inline: true
+			}, {
+				name: 'Guild Roles:',
+				value: `> ${guild.roles.cache.array().filter(r => r.name !== '@everyone').join(', ')}`
```

Comment:
this needs to be refactored to account for there being so many roles that the embed overflows and hits the character limit

also the `@everyone` role has the same ID and the guild, so a better filter would be `role.id !== guild.id` IMO

Reply:
@Dev-Jinx This still needs to be refactored per the first part of my previous comment
> this needs to be refactored to account for there being so many roles that the embed overflows and hits the character limit